### PR TITLE
Fixing broken link

### PR DESCRIPTION
**Pull request by vishalchangrani**

### CODE REVIEW

File: src/pages/index.js [hunk 1]
```diff
@@ -140,7 +140,7 @@ export default function Home() {
                 </p>
                 <p>
                   Cadence includes several language features that prevent entire classes of bugs via a strong static type system, <a href="https://developers.flow.com/cadence/language/functions#function-preconditions-and-postconditions">design by contract</a>,
-                  and <a href="https://developers.flow.com/cadence/language/capability-based-access-control">capability-based access control</a>.
+                  and <a href="https://developers.flow.com/cadence/language/access-control">capability-based access control</a>.
```
Comment (turbolent) [inline src/pages/index.js:143]: The "Access Control" page describes access control functionality of code, e.g. the `access` keyword.

This should link to the capabilities page:

```suggestion
                  and <a href="https://cadence-lang.org/docs/language/capabilities">capability-based access control</a>.
```

**Review by turbolent — COMMENTED** [review]
While at it, can you please update all other links from https://developers.flow.com/cadence to https://cadence-lang.org/docs ?

**Comment by vishalchangrani** [comment]
> While at it, can you please update all other links from https://developers.flow.com/cadence to https://cadence-lang.org/docs ?

done

**Comment by vishalchangrani** [comment]
vercel fails for me :(
`This Deployment has been disabled.`

**Comment by vishalchangrani** [comment]
@turbolent Vercel throws [this](https://vercel.com/onflow/cadence-lang-org/D7aiWLoQzi2skuQWD5hLquRCoUV6?filter=errors) error. would you know why?

**Comment by turbolent** [comment]
@vishalchangrani That's not you, even `main` looks broken locally. Seems like some dependency got updated and it assumes to run in the browser. We'll need to fix this on `main` first

**Comment by turbolent** [comment]
I've opened https://github.com/onflow/cadence-lang.org/issues/176

**Review by turbolent — APPROVED** [review]

**Pull request merged**